README.md

```diff
@@ -0,0 +1,51 @@
+# Kubernetes Setup for Prometheus and Grafana
+
+> Work in progress. Please fix me :)
+
+```bash
+kubectl create --filename prometheus-configmap.yaml
+# kubectl get configmaps
+# kubectl delete configmaps/prometheus
+
+kubectl create --filename prometheus-deployment.yaml
+# kubectl get --all-namespaces --output wide pods
+# kubectl logs prometheus-2556266794-sd260
+# kubectl delete pods/prometheus-2556266794-sd260
+
+kubectl create --filename prometheus-services.yaml
```

Comment:
Usually you create the service first and then fill it with pods from an rc or depl, at least that's what I've seen most also when Kelsey or other Google and CoreOS folks show stuff most of the time IIRC.